Source/Csla/Core/GraphMerger.cs

```diff
@@ -129,27 +129,16 @@ private static void CopyField(object source, object target, string fieldName)
     /// </summary>
     /// <param name="target">Target of merge.</param>
     /// <param name="source">Source for merge.</param>
-    /// <param name="isAsync">Called from sync or async method.</param>
-    private void MergeGraph(IEditableCollection target, IEditableCollection source, bool isAsync = false)
+    private void MergeGraph(IEditableCollection target, IEditableCollection source)
```

Comment:
Thank you for these changes.

I still think we probably need an async `MergeGraph` method that merges collections by awaiting async versions of the `MergeBusinessListXYZ` methods.

Otherwise, if I'm implementing those methods in a collection, how would I invoke async rule checking or other behaviors on my child objects?

Reply:
Ok. let me check.

Reply:
I have done some changes and added MergeGraphAsync method.